uber/models.py

```diff
@@ -1150,7 +1150,7 @@ def _staffing_adjustments(self):
             old_staffing = self.orig_value_of('staffing')
             if self.staffing and not old_staffing or self.ribbon == c.VOLUNTEER_RIBBON and old_ribbon != c.VOLUNTEER_RIBBON:
                 self.staffing = True
-            elif old_staffing and not self.staffing or self.ribbon != c.VOLUNTEER_RIBBON and old_ribbon == c.VOLUNTEER_RIBBON:
+            elif old_staffing and not self.staffing or (self.ribbon != c.DEPT_HEAD_RIBBON and self.ribbon != c.VOLUNTEER_RIBBON and old_ribbon == c.VOLUNTEER_RIBBON):
```

Comment:
Whoops, just realized a stylistic improvement. This could also be written as:
`self.ribbon not in [c.VOLUNTEER_RIBBON, c.DEPT_HEAD_RIBBON]`

Ironically that would also let you remove the parentheses.